content/python/glossaries/built-in-functions/abs.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: "Python Built-In Functions: `abs()`"
```

Comment:
For all the Metadata Titles in Python Glossaries, no need to write "Python Built-In Functions:"

It can just be: "`abs()`"